vmdb/app/models/widget_import_validator.rb

```diff
@@ -1,8 +1,11 @@
 class WidgetImportValidator
+  class NonValidWidgetYamlError < StandardError; end
```

Comment:
Should it be NonValid or Invalid.  The former just sounds weird but maybe there's a nuance there?